fastlane/Fastfile

```diff
@@ -64,7 +64,7 @@ platform :android do
     upload_to_play_store(
       apk: apkPath,
       track: 'production',
-      rollout:, 0.001 # ie. 0.1%
+      rollout: 0.001 # ie. 0.1%
```

Comment:
It's probably `rollout: 0.001, # ie. 0.1%`. See comma at the end.

Reply:
Ah good catch 😅 Updated